lib/gems/pending/appliance_console/database_replication.rb

```diff
@@ -70,6 +76,16 @@ def generate_cluster_name
       true
     end
 
+    def write_pgpass_file
+      File.open(PGPASS_FILE, "w") do |f|
+        f.write("*:*:#{database_name}:#{database_user}:#{database_password}\n")
+        f.write("*:*:replication:#{database_user}:#{database_password}\n")
+      end
+
+      FileUtils.chmod(0600, PGPASS_FILE)
+      FileUtils.chown("postgres", "postgres", PGPASS_FILE)
```

Comment:
@carbonin Looks really good.

Just one Suggestion:
Although this will work it might be best to return true here. It would avoid having a string literal in the condition at the caller.